README.md

```diff
@@ -265,6 +266,49 @@ key_stats.beta # 1.4135449089973444
 
 See [#key-stats](https://iexcloud.io/docs/api/#key-stats) for detailed documentation or [key_stats.rb](lib/iex/resources/key_stats.rb) for returned fields.
 
+### Get Advanced Stats
+
+Fetches company's advanced stats for a symbol
+
+```ruby
+advanced_stats = client.advanced_stats('MSFT')
+
+advanced_stats.total_cash # 66301000000
+advanced_stats.total_cash_dollars # "$66,301,000,000"
+advanced_stats.current_debt # 20748000000
+advanced_stats.current_debt_dollars # "$2,074,8000,000"
+advanced_stats.revenue # 265809000000
+advanced_stats.revenue_dollars # "$265,809,000,000"
+advanced_stats.gross_profit # 101983000000 
+advanced_stats.gross_profit_dollar # "$101,983,000,000"
+advanced_stats.total_revenue # 265809000000
+advanced_stats.total_revenue_dollar # "$265,809,000,000"
+advanced_stats.ebitda # 80342000000
+advanced_stats.ebitda_dollar # "$80,342,000,000"
+advanced_stats.revenue_per_share # 0.02
+advanced_stats.revenue_per_share_dollar # "$0.02"
+advanced_stats.revenue_per_employee # 2013704.55
+advanced_stats.revenue_per_employee_dollar # "$2,013,704.55"
+advanced_stats.debt_to_equity # 1.07
+advanced_stats.profit_margin # 22.396157
+advanced_stats.enterprise_value # 1022460690000
+advanced_stats.enterprise_value_dollar # "$1,022,460,690,000"
+advanced_stats.enterprise_value_to_revenue # 3.85
+advanced_stats.price_to_sales # 3.49
+advanced_stats.price_to_sales_dollar # "$3.49"
+advanced_stats.price_to_book # 8.805916432564608
+advanced_stats.forward_pe_ratio # 18.14
+advanced_stats.pe_high # 22.61
+advanced_stats.pe_low # 11.98
+advanced_stats.avg_30_volume # 
```

Comment:
this doesn't exist here ... remove

let's check all the fields in advanced_stats? did we miss anything else?

Reply:
Yup my bad. Will double check. Hoping to have time to clean this up tomorrow or on the weekend.

Reply:
Removed `avg_30_volume`. While confirming all fields are correct using the actual API call, I noticed `put_call_ratio` was missing so I also added it.